ccmlib/node.py

```diff
@@ -232,29 +232,34 @@ def get_base_cassandra_version(self):
         version = self.get_cassandra_version()
         return float(version[:version.index('.') + 2])
 
-    def set_configuration_options(self, values=None, batch_commitlog=None):
+    def set_configuration_options(self, values=None):
         """
         Set Cassandra configuration options.
         ex:
             node.set_configuration_options(values={
                 'hinted_handoff_enabled' : True,
                 'concurrent_writes' : 64,
             })
-        The batch_commitlog option gives an easier way to switch to batch
-        commitlog (since it requires setting 2 options and unsetting one).
         """
         if values is not None:
             for k, v in iteritems(values):
                 self.__config_options[k] = v
-        if batch_commitlog is not None:
-            if batch_commitlog:
-                self.__config_options["commitlog_sync"] = "batch"
-                self.__config_options["commitlog_sync_batch_window_in_ms"] = 5
-                self.__config_options["commitlog_sync_period_in_ms"] = None
-            else:
-                self.__config_options["commitlog_sync"] = "periodic"
-                self.__config_options["commitlog_sync_period_in_ms"] = 10000
-                self.__config_options["commitlog_sync_batch_window_in_ms"] = None
+
+        self.import_config_files()
+
+    def set_batch_commitlog(self, batch_commitlog=False):
+        """
+        The batch_commitlog option gives an easier way to switch to batch
+        commitlog (since it requires setting 2 options and unsetting one).
+        """
+        if batch_commitlog:
+            self.__config_options["commitlog_sync"] = "batch"
+            self.__config_options["commitlog_sync_batch_window_in_ms"] = 5
+            self.__config_options["commitlog_sync_period_in_ms"] = None
+        else:
+            self.__config_options["commitlog_sync"] = "periodic"
+            self.__config_options["commitlog_sync_period_in_ms"] = 10000
+            self.__config_options["commitlog_sync_batch_window_in_ms"] = None
```

Comment:
This isn't a complaint about the PR, but about the original code: does this need to be duplicated across `cluster` and `node`? Can this PR also refactor the `cluster` implementation to use the `node` implementation?